docs/components/dropdown-menu.md

```diff
@@ -0,0 +1,394 @@
+
+### Properties
+
+- `item` - the properties the router-link is built from, it expects a `label` and a `to`.
+- `disabled` - a boolean (default to `false`), whether or not to disable the item.
+- `selected` - a boolean (default to `false`), whether or not the item is selected.
+- `hasDivider` - a boolean (default to `false`), whether or not the item should have a divider bar displayed above it
+- `isDangerous` - a boolean (default to `false`), whether or not to apply danger styles (text color is red)
+- `selectionMenuChild` - a boolean (default to `false`), whether the parent is a `selectionMenu` or not (used for events)
+
+```html
+<KDropdownItem :item="{ label: 'Leave the page', to: { path: '/' } }" />
+```
+
+There are 3 primary item types:
+
+- `router-link`
+  - the generic type generated using the `items` prop on `KDropdownMenu`
+  - the generic type generated using the `item` prop on `KDropdownItem`
+- `button` - this item is generated if a handler is specified for the `@click` event on a `KDropdownItem`
+- `custom` - no special handling, you completely control the content
+
+<div>
+  <KDropdownMenu label="Variety">
+    <template #items>
+      <KDropdownItem :item="youAreHere" />
+      <KDropdownItem @click="clickHandler">
+        A button
+      </KDropdownItem>
+      <KDropdownItem
+        disabled
+        @click="clickHandler"
+      >
+        Disabled button
+      </KDropdownItem>
+      <KDropdownItem
+        has-divider
+        is-dangerous
+        class="d-inline-block"
+      >
+        <a
+          href="http://www.google.com"
+          rel="noopener"
+          target="_blank"
+        >
+          External link
+          <KIcon
+            icon="externalLink"
+            size="12"
+            class="ml-2"
+          />
+        </a>
+      </KDropdownItem>
+    </template>
+  </KDropdownMenu>
+</div>
+
+```html
+<KDropdownMenu label="Variety">
+  <template #items>
+    <KDropdownItem :item="{ label: 'You are here', to: { path: '/components/dropdown-menu.html' } }" />
+    <KDropdownItem @click="clickHandler">
+      A button
+    </KDropdownItem>
+    <KDropdownItem
+      disabled
+      @click="clickHandler"
+    >
+      Disabled button
+    </KDropdownItem>
+    <KDropdownItem
+      has-divider
+      is-dangerous
+      class="d-inline-block"
+    >
+      <a
+        href="http://www.google.com"
+        rel="noopener"
+        target="_blank"
+      >
+        External link
+        <KIcon
+          icon="externalLink"
+          size="12"
+          class="ml-2"
+        />
+      </a>
+    </KDropdownItem>
+  </template>
+</KDropdownMenu>
+```
+
+### Events
+
+| Event     | Description             |
+| :-------- | :------------------ |
+| `clicked` | Fires when a menu item is clicked |
+| `changed` | Fires when items with `selectionMenuChild` prop are clicked; returns `selectedItem` Object or null |
+
```

Comment:
We also need to document the `toggleDropdown` event

Reply:
Fixed